cluster_test.spec.ts

```diff
@@ -1,5 +1,3 @@
-"use strict";
```

Comment:
Do these changes impact `tsc` compilation errors?

Reply:
Nope, they appear when you turn on strict mode globally (tickets coming soon, still sifting through the errors)